File: 1329-minimum-cost-to-move-chips-to-the-same-position/solution.py
class Solution:
    def minCostToMoveChips(self, position: List[int]) -> int:
        # counter = collections.Counter(position)
        
        cnt = 0
        for pos in position:
            if pos % 2 == 0:
                cnt += 1
        return min(cnt, len(position)-cnt)

File: 1329-minimum-cost-to-move-chips-to-the-same-position/solution.cpp
class Solution {
public:
    int minCostToMoveChips(vector<int>& position) {
        int even = 0, odd = 0;
        for (int pos : position) {
            (pos % 2 == 1) ? ++odd : ++even;
        }
        return min(odd, even);
    }
};